api/src/test/java/org/pmiops/workbench/testconfig/fixtures/ReportingUserFixture.java

```diff
@@ -56,9 +56,9 @@
       Timestamp.from(Instant.parse("2015-05-11T00:00:00.00Z"));
   public static final String USER__ACCESS_TIER_SHORT_NAMES =
       AccessTierService.REGISTERED_TIER_SHORT_NAME;
-  public static final Timestamp USER__DATA_USE_AGREEMENT_BYPASS_TIME =
+  public static final Timestamp USER__DATA_USER_CODE_OF_CONDUCT_AGREEMENT_BYPASS_TIME =
       Timestamp.from(Instant.parse("2015-05-14T00:00:00.00Z"));
-  public static final Timestamp USER__DATA_USE_AGREEMENT_COMPLETION_TIME =
+  public static final Timestamp USER__DATA_USER_CODE_OF_CONDUCT_COMPLETION_TIME =
       Timestamp.from(Instant.parse("2015-05-15T00:00:00.00Z"));
   public static final Integer USER__DATA_USER_CODE_OF_CONDUCT_SIGNED_VERSION = 11;
```

Comment:
nit: why do we have 2 _ and not  USER_DATA_USER_CODE_OF_CONDUCT_SIGNED_VERSION

Reply:
This is a weird file where 2 __ is the standard